flixel/FlxCamera.hx

```diff
@@ -1638,105 +1398,60 @@ class FlxCamera extends FlxBasic
 	/**
 	 * Fill the camera with the specified color.
 	 *
-	 * @param   Color        The color to fill with in `0xAARRGGBB` hex format.
-	 * @param   BlendAlpha   Whether to blend the alpha value or just wipe the previous contents. Default is `true`.
+	 * @param   color        The color to fill with in `0xAARRGGBB` hex format.
+	 * @param   blendAlpha   Whether to blend the alpha value or just wipe the previous contents. Default is `true`.
 	 */
-	public function fill(Color:FlxColor, BlendAlpha:Bool = true, FxAlpha:Float = 1.0, ?graphics:Graphics):Void
+	@:deprecated("The 4-arg fill(color, blendAlpha, fxAlpha, ?graphics) is deprecated, use the 2-arg fill(color, blendAlpha) instead.")
+	overload public inline extern function fill(color:FlxColor, blendAlpha:Bool = true, fxAlpha:Float = 1.0, ?graphics:Graphics):Void
 	{
-		if (FlxG.renderBlit)
-		{
-			if (BlendAlpha)
-			{
-				_fill.fillRect(_flashRect, Color);
-				buffer.copyPixels(_fill, _flashRect, _flashPoint, null, null, BlendAlpha);
-			}
-			else
-			{
-				buffer.fillRect(_flashRect, Color);
-			}
-		}
-		else
-		{
-			final targetGraphics = (graphics == null) ? canvas.graphics : graphics;
-
-			targetGraphics.overrideBlendMode(null);
-			targetGraphics.beginFill(Color, FxAlpha);
-			// i'm drawing rect with these parameters to avoid light lines at the top and left of the camera,
-			// which could appear while cameras fading
-			targetGraphics.drawRect(viewMarginLeft - 1, viewMarginTop - 1, viewWidth + 2, viewHeight + 2);
-			targetGraphics.endFill();
-		}
+		color.alphaFloat = fxAlpha;
+		view.fill(color, blendAlpha);
+	}
+
+	/**
+	 * Fill the camera with the specified color.
+	 *
+	 * @param   color        The color to fill with in `0xAARRGGBB` hex format.
+	 * @param   blendAlpha   Whether to blend the alpha value or just wipe the previous contents. Default is `true`.
+	 */
+	overload public inline extern function fill(color:FlxColor, blendAlpha:Bool = true):Void
```

Comment:
Rather than overloading maybe we should define a new fillView() or use view.fill directly?